Android/app/src/main/java/io/github/project_travel_mate/travel/ShoppingCurrentCityActivity.java

```diff
@@ -195,14 +198,15 @@ public void onResponse(Call call, final Response response) {
                                 Utils.hideKeyboard(ShoppingCurrentCityActivity.this);
                             }
                             animationView.setVisibility(View.GONE);
+                            layout.setVisibility(View.VISIBLE);
                             textView.setVisibility(View.GONE);
                             lv.setAdapter(new ShoppingAdapter(ShoppingCurrentCityActivity.this, feedItems));
                         } catch (JSONException | IOException e) {
                             e.printStackTrace();
                             networkError();
                         }
                     } else {
-                        networkError();
+                        noResults();
```

Comment:
The code will reach here in case of network error only.

Reply:
No I tested it. It's functioning perfectly. You can test it.